docs/how-to-guides/certificate-rotation.md

```diff
@@ -43,6 +43,14 @@ The date in the `notAfter` line is the expiration date of the cluster CA certifi
 
 ## Step 2: Rotate certificates
 
+Lokoctl transfers the newly generated certificates to the controller nodes over SSH. Run the
+following command to load the ssh key:
```

Comment:
```suggestion
Lokoctl transfers the newly generated certificates to the controller nodes over SSH. Run the
following command to load the one of the SSH keys specified in the cluster configuration (in this case `~/.ssh/id_rsa`):
```

Reply:
Maybe also add a note mentioning that on Equinix Metal you need to have the public IP from the machine where you run lokoctl added to the `management_cidrs` argument in the cluster configuration (I ran into this issue when testing :D).

Reply:
Done, PTAL.